admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,253 @@
+
+To run the tool on the command line:
+
+----
+$ cluster-capacity --kubeconfig <path-to-kubeconfig> \
+    --podspec <path-to-pod-spec>
+----
+
+The `--kubeconfig` option indicates your Kubernetes configuration file,
+and the `--podspec` option indicates a sample pod specification file, which
+the tool uses for estimation. The `podspec` specifies its resource requirements
+as `limits` or `requests`. The cluster capacity tool takes the pod's resource
+requirements into account for its estimation analysis.
+
+An example of the pod specification input is:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: small-pod
+  labels:
+    app: guestbook
+    tier: frontend
+spec:
+  containers:
+  - name: php-redis
+    image: gcr.io/google-samples/gb-frontend:v4
+    imagePullPolicy: Always
+    resources:
+      limits:
+        cpu: 150m
+        memory: 100Mi
+      requests:
+        cpu: 150m
+        memory: 100Mi
+----
+
+The `--verbose` option can also be provided as part
+of the above command to output a detailed description of how many pods can be
+scheduled on each node in the cluster:
+
+----
+$ cluster-capacity --kubeconfig <path-to-kubeconfig> \
+    --podspec <path-to-pod-spec> --verbose
+----
+
+The output will look similar to the following:
+
+----
+small-pod pod requirements:
+	- CPU: 150m
+	- Memory: 100Mi
+
+The cluster can schedule 52 instance(s) of the pod small-pod. <1>
+
+Termination reason: Unschedulable: No nodes are available that match all of the
+following predicates:: Insufficient cpu (2).
+
+Pod distribution among nodes:
+small-pod
+	- 192.168.124.214: 26 instance(s)
+	- 192.168.124.120: 26 instance(s)
+----
+<1> The number of estimated pods that can be scheduled on the cluster. 
+
+[[admin-guide-running-cluster-capacity-inside-pod]]
+== Running Cluster Capacity as a Job Inside of a Pod
+
+Running the cluster capacity tool as a job involves using a `ConfigMap`.
+
+. Create the cluster role:
++
+----
+$ cat << EOF| oc create -f -
+kind: ClusterRole
+apiVersion: v1
+metadata:
+  name: cluster-capacity-role
+rules:
+- apiGroups: [""]
+  resources: ["pods", "nodes", "persistentvolumeclaims", "persistentvolumes", "services"]
+  verbs: ["get", "watch", "list"
+EOF
+----
+
+. Create the service account:
++
+----
+$ oc create sa cluster-capacity-sa
+----
+
+. Add the role to the service account:
++
+----
+$ oadm policy add-cluster-role-to-user cluster-capacity-role \
+    system:serviceaccount:default:cluster-capacity-sa
+----
+
+. Define and create the pod specification:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: small-pod
+  labels:
+    app: guestbook
+    tier: frontend
+spec:
+  containers:
+  - name: php-redis
+    image: gcr.io/google-samples/gb-frontend:v4
+    imagePullPolicy: Always
+    resources:
+      limits:
+        cpu: 150m
+        memory: 100Mi
+      requests:
+        cpu: 150m
+        memory: 100Mi
+----
+
+. Make Pod Specification Available
```

Comment:
Its same as your point 4, so not required here. I dont remember if Derek referred to something else.